operator/internal/status/lokistack.go

```diff
@@ -44,6 +45,11 @@ var (
 		Message: messageDegradedNodeLabels,
 		Reason:  string(lokiv1.ReasonNoZoneAwareNodes),
 	}
+	conditionTopologyKeyNodeLabelsEmpty = metav1.Condition{
```

Comment:
I'd use the `conditionDegraded` prefix here as well:

```suggestion
	conditionDegradedEmptyNodeLabel = metav1.Condition{
```